src/common/rand/rand.c

```diff
@@ -54,9 +64,14 @@ OQS_API void OQS_randombytes_custom_algorithm(void (*algorithm_ptr)(uint8_t *, s
 }
 
 OQS_API void OQS_randombytes(uint8_t *random_array, size_t bytes_to_read) {
+	if (oqs_randombytes_algorithm == NULL) {
+		fprintf(stderr, "OQS_randombytes called before algorithm was set\n");
```

Comment:
Well, possibly not before -- might also have happened because someone switched to "system" (see above), right?

Reply:
I updated the error message to make it more “general”.